### PR TITLE
Move to packer

### PR DESCRIPTION
**Pull request by ArendJan**
Pull request https://github.com/mirte-robot/mirte-install-scripts/pull/41 must first be merged.
This will change the build system to packer
Uses workflows to build the 3 images and releases them on release.
Also adds all the other repositories to the release.

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+jobs:
+    build-and-push:
+        # container:
+        #   image: ubuntu:22.04
+        #   options: --privileged
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                image: [
+                        mirteopi,
```
Comment (mklomp) [inline .github/workflows/build.yaml:17]: By default we will only release opizero2, and raspberry pi. We should not by default include orangepizero1. And in doubt about 3b.

File: add_partition_local/default_partition_files/machine_config.yaml
Comment (mklomp) [inline add_partition_local/default_partition_files/machine_config.yaml:1]: Replace these with dummy values

File: default_aj_repos.yaml
Comment (mklomp) [inline default_aj_repos.yaml:1]: This file can be removed.

File: main_repos.yaml
Comment (mklomp) [inline main_repos.yaml:1]: what is the difference between main_repos.yaml and default_repos.yaml?

File: mirte_main_install.sh
Comment (mklomp) [inline mirte_main_install.sh:1]: Tha file could have a bit more comments on why the commands are needed.

File: wheels/numpy-1.24.4-cp38-cp38-linux_armv7l.whl
Comment (mklomp) [inline wheels/numpy-1.24.4-cp38-cp38-linux_armv7l.whl:1]: Why is this one here?

Comment (ArendJan) [reply wheels/numpy-1.24.4-cp38-cp38-linux_armv7l.whl:1]: it takes ages to compile numpy for the orange pi zero 1, this is a precompiled package to speed up image generation(>5-10min)

File: default_aj_repos.yaml
Comment (ArendJan) [reply default_aj_repos.yaml:1]: removed

File: main_repos.yaml
Comment (ArendJan) [reply main_repos.yaml:1]: nothing, had one for the ci and one for local dev. main_repos is removed

File: .github/workflows/build.yaml (hunk 1 above)
Comment (ArendJan) [reply .github/workflows/build.yaml]: I've removed zero1, changed the naming and currently testing rpi4b

Comment (ArendJan) [reply .github/workflows/build.yaml]: Added rpi4b, unable to test it

**Review by mklomp — APPROVED** [review]

**Pull request merged**